cmd/clusterctl/cmd/version_checker.go

```diff
@@ -170,6 +194,40 @@ func (v *versionChecker) getLatestRelease() (*ReleaseInfo, error) {
 	return &vs.LatestRelease, nil
 }
 
+func (v *versionChecker) goproxyGetLatest() (*ReleaseInfo, error) {
+	if v.goproxyClient == nil {
+		return nil, nil
+	}
+
+	gomodulePath := path.Join("github.com", "kubernetes-sigs", "cluster-api")
```

Comment:
Just curious, why would the Go module path not be `sigs.k8s.io/cluster-api`?

Reply:
Both should work. But maybe we should prefer `sigs.k8s.io/cluster-api` here?

Reply:
done